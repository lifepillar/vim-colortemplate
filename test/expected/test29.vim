" Name:         Test 29
" Author:       y
" Maintainer:   w
" License:      Vim License (see `:help license`)

set background=dark

hi clear
let g:colors_name = 'test29'

let s:t_Co = exists('&t_Co') && !has('gui_running') ? (&t_Co ?? 0) : -1

hi Normal guifg=#ffffff guibg=#000000 gui=NONE cterm=NONE

if s:t_Co >= 256
  hi Normal ctermfg=255 ctermbg=16 cterm=NONE
  unlet s:t_Co
  finish
endif

" vim: et ts=2 sw=2
