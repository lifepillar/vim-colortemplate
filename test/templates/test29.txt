Colortemplate Options: creator=0 quiet=1 source_comment=0 timestamp=0 warnings=0
Full name:Test 29
Short name:test29
Author:y
Maintainer:w
documentation
  @shortname_transp_bg <-- this is ok (interpolated)
  @shortnametransp_bg  <-- ditto
  @suppress            <-- this is ignored (not interpolated)
enddocumentation
Background:dark
Color:black rgb(0,0,0) 16 Black
Color:white #ffffff 255 White
Normal white black

; vim: ft=colortemplate
