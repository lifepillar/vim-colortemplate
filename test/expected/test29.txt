  test29_transp_bg <-- this is ok (interpolated)
  test29transp_bg  <-- ditto
  @suppress            <-- this is ignored (not interpolated)
